include/sensors/core/tasks/capacitive_driver.hpp

```diff
@@ -371,10 +359,46 @@ class FDC1004 {
     bool echoing = false;
     bool bind_sync = false;
     bool max_capacitance_sync = false;
-    std::bitset<2> data_stable{"00"};
     std::array<uint16_t, 2> baseline_results{};
     std::array<uint16_t, 2> polling_results{};
 
+    auto send_followup(i2c::messages::TransactionResponse &m) -> void {
+        auto address = utils::reg_from_id<uint8_t>(m.id.token);
+        uint8_t followup_address = 0;
+        auto tags = utils::tags_from_token(m.id.token);
+        i2c::messages::TransactionIdentifier id = m.id;
+        if (address == static_cast<uint8_t>(fdc1004::Registers::FDC_CONF)) {
```

Comment:
kind of a nit but i like this kind of thing a bit more as a `switch` because it lines up nicer